doc/source/user_guide/video.txt

```diff
@@ -97,3 +97,12 @@ videos that exceed available RAM. It works on all floats if FFmpeg is installed.
     from moviepy.editor import VideoFileClip
     myclip = VideoFileClip("some_video.avi")
 
+Imageio
+^^^^^^^^
+
+`Imageio <http://imageio.github.io/>`_ offers a way to load video using FFmpeg. It's quite similar to moviepy.
+
+ .. code-block:: python
```

Comment:
I think this has to be indented three spaces to render properly. There should also be a blank line between `.. code-block:: python` and the code.